verticapy/connect.py

```diff
@@ -142,25 +132,98 @@ def change_auto_connection(name: str):
         f.close()
     else:
         raise NameError(
-            "The input name is incorrect. The connection '{}' has never been created.\nUse the new_auto_connection function to create a new connection.".format(
+            "The input name is incorrect. The connection '{}' has never been created.\nUse the new_connection function to create a new connection.".format(
                 name
             )
         )
 
+# ---#
+def close_connection():
+    """
+---------------------------------------------------------------------------
+Close the Database connection.
+    """
+    if verticapy.options["connection"]["conn"]:
+        try:
+            verticapy.options["connection"]["conn"].close()
+        except:
+            pass
```

Comment:
Why `pass`? Isn't better to print an error message?

Reply:
I did a new PR to remove all the "try" in the connect.py file.